gnat2goto/ireps/follow.adb

```diff
@@ -5,6 +5,11 @@ package body Follow is
       while Next /= 0 and then Kind (Next) = I_Symbol_Type loop
          Next := ST (Intern (Get_Identifier (Next))).SymType;
       end loop;
+      if Kind (Next) = I_Bounded_Signedbv_Type then
+         Next :=
+           Make_Signedbv_Type (I_Subtype => Get_Subtype (Next),
+                               Width     => Get_Width (Next));
+      end if;
```

Comment:
As I recall the problem being solved here is not at all obvious. We strip the type of a bounded integer (to make it a simple integer) upon export - to sidestep a cbmc invariant on assignment that rhs and lhs type must be equal. 
Some hint at to the issue being solved by this code block would be useful for future users/ team members.

Reply:
Commented, but not here: the implementation is now generated in the ireps py-script and called from `driver.adb` (where it's also documented).